<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorSessionForm</class>
 <widget class="QWidget" name="DoctorSessionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生会诊列表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Cards Container -->
   <item>
    <widget class="QWidget" name="cardsWidget">
     <layout class="QVBoxLayout" name="cardsLayout">

      <!-- Session Card -->
      <item>
       <widget class="QFrame" name="sessionCard">
        <property name="styleSheet">
         <string notr="true">
          QFrame { background: #ffffff; border-radius:16px; padding:16px; }
          QLabel { font-size:14px; color:#374151; background:transparent; margin:0px; padding:0px; }
          QLabel#sessionHeader { font-size:16px; font-weight:bold; margin-bottom:12px; }
          QLineEdit { border:1px solid #d1d5db; border-radius:8px; padding:6px; font-size:14px; background:#f9fafb; }
          QLineEdit:focus { border:1px solid #3b82f6; background:#ffffff; }
          QLineEdit, QCheckBox { color: black; }
         </string>
        </property>
        <layout class="QVBoxLayout" name="sessionLayout">
         <item>
          <widget class="QLabel" name="sessionHeader">
           <property name="text"><string>会诊信息</string></property>
          </widget>
         </item>
         <item>
         <layout class="QFormLayout" name="sessionFormLayout">
          <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
          <property name="labelAlignment">
           <set>Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeft|Qt::AlignTop</set>
          </property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelSessionTime">
            <property name="text"><string>时间：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="sessionTimeEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
         </layout>
         </item>
        </layout>
       </widget>
      </item>

      <!-- Patient Card -->
      <item>
       <widget class="QFrame" name="patientCard">
        <property name="styleSheet">
         <string notr="true">
          QFrame { background:#ffffff; border-radius:16px; padding:16px; }
          QLabel { font-size:14px; color:#374151; background:transparent; margin:0px; padding:0px; }
          QLabel#patientHeader { font-size:16px; font-weight:bold; margin-bottom:12px; }
          QLineEdit { border:1px solid #d1d5db; border-radius:8px; padding:6px; font-size:14px; background:#f9fafb; }
          QLineEdit:focus { border:1px solid #3b82f6; background:#ffffff; }
          QLineEdit, QCheckBox { color: black; }
         </string>
        </property>
        <layout class="QVBoxLayout" name="patientLayout">
         <item>
          <widget class="QLabel" name="patientHeader"><property name="text"><string>患者信息</string></property></widget>
         </item>
         <item>
         <layout class="QFormLayout" name="patientFormLayout">
          <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
          <property name="labelAlignment">
           <set>Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeft|Qt::AlignTop</set>
          </property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelPatientName">
            <property name="text"><string>姓名：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="patientNameEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="labelPatientSex">
            <property name="text"><string>性别：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="patientSexEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QLabel" name="labelPatientAge">
            <property name="text"><string>年龄：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="patientAgeEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="3" column="0">
           <widget class="QLabel" name="labelPatientPhone">
            <property name="text"><string>手机号：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="patientPhoneEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
         </layout>
         </item>


<!-- Centered No Sessions Label -->
<item>
 <spacer>
  <property name="orientation">
   <enum>Qt::Orientation::Vertical</enum>
  </property>
 </spacer>
</item>
<item>
 <widget class="QLabel" name="noSessionsLabel">
  <property name="text">
   <string>当前没有会诊记录</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="visible">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel { font-size:16px; color:#6b7280; }</string>
  </property>
 </widget>
</item>
<item>
 <spacer>
  <property name="orientation">
   <enum>Qt::Orientation::Vertical</enum>
  </property>
 </spacer>
</item>
         <item>
           <widget class="QPushButton" name="messageButton">
             <property name="styleSheet">
               <string notr="true">
                 QPushButton {
                   background: #d1d5db;
                   color: #111827;
                   border-radius: 8px;
                   padding: 10px;
                   font-weight: bold;
                 }
                 QPushButton:hover:!disabled {
                   background-color: #9ca3af;
                 }
                 QPushButton:disabled {
                   background-color: #e5e7eb;
                   color: #6b7280;
                 }
               </string>
             </property>
             <property name="text">
               <string>发送消息</string>
             </property>
           </widget>
         </item>
        </layout>
       </widget>
      </item>

      <!-- End Session Button -->
      <item>
       <widget class="QPushButton" name="endSessionButton">
        <property name="styleSheet">
         <string notr="true">
          QPushButton { background-color: #ef4444; color: white; border-radius: 8px; padding: 10px; font-weight: bold; }
          QPushButton:hover { background-color: #b91c1c; }
         </string>
        </property>
        <property name="text"><string>结束该会诊</string></property>
       </widget>
      </item>

      <!-- Navigation -->
      <item>
        <layout class="QHBoxLayout" name="navLayout">
          <item>
            <widget class="QPushButton" name="prevButton">
              <property name="styleSheet">
                <string notr="true">
                  QPushButton {
                    background: #d1d5db;
                    color: #111827;
                    border-radius: 8px;
                    padding: 10px;
                    font-weight: bold;
                  }
                  QPushButton:hover:!disabled {
                    background-color: #9ca3af;
                  }
                  QPushButton:disabled {
                    background-color: #e5e7eb;
                    color: #6b7280;
                  }
                </string>
              </property>
              <property name="text">
                <string>上一页</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="nextButton">
              <property name="styleSheet">
                <string notr="true">
                  QPushButton {
                    background: #d1d5db;
                    color: #111827;
                    border-radius: 8px;
                    padding: 10px;
                    font-weight: bold;
                  }
                  QPushButton:hover:!disabled {
                    background-color: #9ca3af;
                  }
                  QPushButton:disabled {
                    background-color: #e5e7eb;
                    color: #6b7280;
                  }
                </string>
              </property>
              <property name="text">
                <string>下一页</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>

     </layout>
    </widget>
   </item>

   <!-- Bottom Spacer -->
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
     <property name="sizeHint" stdset="0">
      <size><width>20</width><height>40</height></size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
